### ISSUE TITLE
GRC: invalid python generated when using gr-submodule name as flowgraph ID

### ISSUE THREAD

**Issue by marcusmueller** [gnuradio/gnuradio#7102]
### What happened?

When you use e.g. `qtgui` as ID for your flowgraph, things go south. You can generate code then, but it's usually impossible to run – because your block class name conflicts with an import.

### System Information

latest source gtk-grc, Py3.11

### GNU Radio Version

3.11-git (main)

### Specific Version

0.0.0.0

### Steps to Reproduce the Problem

1. new fg
2. set id in Options to `qtgui`
3. generate
4. try to run

### Relevant log output

```shell
module qtgui has no member utils
```

**Comment by willcode** [comment]
There are so many ways to make naming conflicts! Unless we start adding pre/suffixes to names, this gets to be whack-a-mole.

**Comment by marcusmueller** [comment]
yeah, I think the appropriate reaction can only be to check for every import whether it introduces a conflict, ideally at the point where you add the import (or change any block/fg id), but latest at the generation stage

**Comment by marcusmueller** [comment]
note that the logic to do such things is there – can't have e.g. two variables with the same id!
